lua/mapvote/server/modules/rtv.lua

```diff
@@ -1,5 +1,5 @@
-MapVote.RTV = MapVote.RTV or {}
-local RTV = MapVote.RTV
+local RTV = MapVote.RTV or {}
```

Comment:
lua_ls works better if its like this for some reason

Reply:
Yeah that bugs me - ran into the same issue before